edu-services/gradebook-service/api/src/java/org/sakaiproject/service/gradebook/shared/GradebookService.java

```diff
@@ -564,6 +564,7 @@ public boolean isExternalAssignmentDefined(String gradebookUid,
 
 	public Map getImportCourseGrade(String gradebookUid);
 	public Map getImportCourseGrade(String gradebookUid, boolean useDefault);
+	public Map getImportCourseGrade(String gradebookUid, boolean useDefault, boolean mapTheGrades);
```

Comment:
Just noticed no javadocs explaining what this does. I'll migrate them from the impl up to the interface so they are visible. I have another PR coming in this area anyway.